scripts/tag-reconciler/tag-reconciler.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"fmt"
+	"slices"
+
+	"github.com/cri-o/cri-o/internal/version"
+	"github.com/cri-o/cri-o/scripts/utils"
+	"github.com/sirupsen/logrus"
+	"sigs.k8s.io/release-sdk/git"
+	"sigs.k8s.io/release-utils/command"
+	"sigs.k8s.io/release-utils/env"
+)
+
+func main() {
+	logrus.SetFormatter(&logrus.TextFormatter{DisableTimestamp: true})
+	if err := run(); err != nil {
+		logrus.Fatalf("Unable to run: %v", err)
+	}
+}
+
+func run() error {
+	if !env.IsSet(utils.GithubTokenEnvKey) {
+		return fmt.Errorf(
+			"environment variable %q is not set", utils.GithubTokenEnvKey,
+		)
```

Comment:
I think this is not really a very long line? :smile: 

```suggestion
		return fmt.Errorf("environment variable %q is not set", utils.GithubTokenEnvKey)
```